test/xspec-eqname.xspec

```diff
@@ -114,7 +113,8 @@
 			select="/Q{}variable-child"><variable-child /></x:variable>
 		<!-- The following definition illustrates redefining a variable and mixing notations
 			for its name. The file xspec-variable.xspec would be a good place to test that,
-			except that that file does not use xslt-version="3.0". -->
+			except that that file did not use xslt-version="3.0".
+			TODO: The default @xslt-version has been changed. Consider relocating the test. -->
```

Comment:
@AirQuick , the original comment was mine. Would you like me to take care of the TODO item on this line and in `xspec-variable.xspec`, after you merge this pull request?

Reply:
Yes, much appreciated!

Reply:
Will do. I created issue #927 for myself.